packages/lesswrong/server/ckEditorToken.ts

```diff
@@ -20,7 +21,7 @@ addStaticRoute('/ckeditor-token', async ({ query }, req, res, next) => {
 
   const user = await getUserFromReq(req)
   const post = await Posts.findOne(documentId)
-  const canEdit = post && Posts.canEdit(user, post)  
+  const canEdit = post && postCanEdit(user, post)  
```

Comment:
Aren't these `canEdit` `canRead` and `canDelete` function used in a bunch of internal Collection specific Vulcan code?

Reply:
Nope, I am wrong, apparently it isn't

Reply:
I did a search on `canEdit`, and it doesn't have any accesses like that (there are a few variables that pun it, but no members or member accesses). `checkAccess`, on the other hand, is vulcan-internal in the way you're thinking, as is very-similarly-named `canUpdate`.

Reply:
Ah, cool. I was probably thinking of `canUpdate`